www/server.js

```diff
@@ -123,8 +123,8 @@ server.route({
     ;
 
     var requestObject = {
-      to: 'hello@therefore.ca',
```

Comment:
@ArsenYun not sure why this setting got changed, but should remain as `hello@`. Also, not seeing the `www/index.html` updated. If you run `npm run serve` it should build and git add the changed www folder ready to go. (note needs node version 4 sadly, it's an old package suite). Thank you!